pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/design/MutableStaticState.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.design;
+
+import net.sourceforge.pmd.testframework.PmdRuleTst;
+
+public class MutableStaticState extends PmdRuleTst {
```

Comment:
Please rename this class that it ends with the suffix "Test" - otherwise the tests won't be executed.